packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -1244,6 +1250,26 @@ export function attach(
       recordMount(fiber, parentFiber);
     }
 
+    if (traceUpdatesEnabled) {
+      const elementType = getElementTypeForFiber(fiber);
+      if (traceNearestHostComponentUpdate) {
+        // If an ancestor updated, we should mark the nearest host nodes for highlighting.
+        if (elementType === ElementTypeHostComponent) {
+          traceUpdatesForNodes.add(fiber.stateNode);
+          traceNearestHostComponentUpdate = false;
+        }
+      } else {
+        if (
+          elementType === ElementTypeFunction ||
+          elementType === ElementTypeClass ||
+          elementType === ElementTypeContext
+        ) {
+          // Otherwise if this is a traced ancestor, flag for the nearest host descendant(s).
+          traceNearestHostComponentUpdate = true;
```

Comment:
This is a bit odd. If there are two nested HostComponents then only the outer one is marked but as soon as there's  an intermediate function it is not?

What's the expected effect for a mounted subtree? Should only the inserted Host node be highlighted or is it every node in the subtree?

Reply:
I think this is the desired behavior- that we only highlight the _nearest_ host components.

This is how the old feature worked. If a composite component or context type updated, we highlighted on the nearest node:
https://github.com/facebook/react-devtools/blob/d839081f79f681617df7584a8ffd2f4163fa40b9/plugins/TraceUpdates/TraceUpdatesBackendManager.js#L59-L66

Reply:
That's for updates but this is the mount path.

Reply:
My mistake. I misunderstood what you were saying. I agree that I should not flip this to true.